verl/trainer/config/ppo_trainer.yaml

```diff
@@ -88,6 +91,15 @@ data:
     # The name of the dataset class within the specified file.
     name: null
 
+  # settings related to curriculum learning
+  curriculum_sampler:
+
+    # the path to the module containing a curriculum class which implements the AbstractCurriculumSampler interface
```

Comment:
please add example values to this field. In other custom configs file paths are used, but it seems here it refers to a module path. Having an example helps aovid the confusion. 
Furthermore, we should probably have a single util function that address both loading a class from a module `a.b.c` and from a file `a/b/c.py`?

Reply:
Initially, I tried implementing this with the existing `load_extern_type`, but the `isinstanceof(RandomCurriculumSampler, AbstractCurriculumSampler)` failed when pointed at the existing `Random` class. I think the way the current `load_extern_type` works it only loads a file in isolation, removing information about its relationships to the rest of the modules. 

If we were to allow users to supply file paths instead of module paths they would be unable to ever subclass `AbstractCurriculumSampler` in a way that would be recognized. I think that it's better to leave the utilities separate and prevent the user from importing from a path to avoid this situation.